modules/Contacts/views/view.detail.php

```diff
@@ -53,6 +53,17 @@ class ContactsViewDetail extends ViewDetail
 	public function display(){
 		global $sugar_config;
 
+		// Fix for issue 1506: removed picture is still cached in the Detail View.
```

Comment:
yes it would be better to keep it separate, so we can pull it in, with any other fix getting in the way